core/src/main/scala/jms4s/JmsTransactedConsumer.scala

```diff
@@ -156,38 +77,31 @@ object JmsTransactedConsumer {
         .compile
         .drain
 
-  private[jms4s] class JmsTransactedConsumerPool[F[_]: Concurrent: ContextShift](pool: Queue[F, JmsResource[F]]) {
+  private[jms4s] class JmsTransactedConsumerPool[F[_]: Concurrent: ContextShift](
+    pool: Queue[F, (JmsContext[F], JmsMessageConsumer[F])]
+  ) {
 
-    val receive: F[Received[F]] =
+    def receive: F[Received[F]] =
```

Comment:
why not a `val` anymore?

Reply:
it could be a val.
I put Concurrent there before, forgot to change it